package maidsafe.vault.protobuf;
option optimize_for = LITE_RUNTIME;

message PmidRecord {
  required bytes pmid_name = 1;
  required int64 stored_count = 2;
  required int64 stored_total_size = 3;
  required int64 lost_count = 4;
  required int64 lost_total_size = 5;
}

message DataElements {
  required bytes name = 1;
  required int32 size = 2;
  optional int32 version = 3;
}

message PmidAccount {
  required PmidRecord pmid_record = 1;
  repeated DataElements data_stored = 2;
}
